src/main/java/io/dockstore/provision/S3CmdPlugin.java

```diff
@@ -217,20 +240,25 @@ private int executeConsoleCommand(String command, boolean printStdout) {
                     try {
                         final BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream()));
                         String line;
+                        boolean carriage = false;
                         while ((line = reader.readLine()) != null) {
                             if (printStdout) {
-                                // The first line of s3cmd plugin will start with "download", must isolate from others
-                                Pattern pattern = Pattern.compile("download.*");
-                                Matcher matcher = pattern.matcher(line);
-                                if (matcher.matches()) {
-                                    System.out.println(line);
+                                // The line prior to the carriage return will start with "download" or "upload", must isolate from others
+                                if (carriage == false) {
+                                    // 's3cmd info' will only display in dockstore --debug mode
+                                    if (command.contains("info")) {
+                                        LOG.debug(line);
```

Comment:
Just leave `LOG.debug(line)` on all the time and let the logging framework deal with this.

Reply:
Currently it puts the s3cmd info output in debug, and everything else in standard output.  Leaving it in all the time will end up with double the output for all non-info commands.